pytezos/operation/group.py

```diff
@@ -202,6 +223,11 @@ def autofill(
         if not OperationResult.is_applied(opg_with_metadata):
             raise RpcError.from_errors(OperationResult.errors(opg_with_metadata))
 
+        # FIXME: Counter has been just incremented in fill(), dirty as fuck
+        self.context.counter -= 1
```

Comment:
There can be more than one content in the operation group.
As an alternative you can reset `context.counter` to `None`, but that will require one extra RPC call.